src/momento/internal/aio/_scs_data_client.py

```diff
@@ -154,7 +155,7 @@ def __init__(self, configuration: Configuration, credential_provider: Credential
         self._endpoint = endpoint
 
         default_deadline: timedelta = configuration.get_transport_strategy().get_grpc_configuration().get_deadline()
-        self._default_deadline_seconds = int(default_deadline.total_seconds())
+        self._default_deadline_seconds = default_deadline.total_seconds()
```

Comment:
grpc's timeout takes fractional seconds to account for milliseconds, and `timedelta.total_seconds` returns fractional seconds, so we needn't truncate